assets/data/cards_pt-br.json

```
@@ -1110,6 +1138,15 @@
       ],
       "description": "É um campo da inteligência artificial que estuda como os computadores podem obter e processar informações de estímulos visuais. Ela se concentra no desenvolvimento de sistemas que podem analisar e entender imagens e vídeos, permitindo que os computadores reconheçam objetos, pessoas, lugares, movimentos e outros elementos visuais. A visão computacional (do inglês, computer vision ou CV) é usada em uma ampla variedade de aplicações, como reconhecimento facial, carros autônomos, diagnóstico médico, entre outros."
     },
+    {
+      "title": "Vue.js",
+      "tags": [
+        "Conceito",
+        "Front-end",
+        "Framework"
+      ],
+      "description": "Vue.js é um framework JavaScript para criar interfaces de usuário. Ele é progressivo, fácil de integrar em projetos e muito utilizado para construir aplicações de página única (SPAs)."
```

Deixei o termo "progressivo" mais implícito, focando na facilidade de integração e no uso em SPAs. O que acha?

```suggestion
      "description": "Vue.js é uma ferramenta JavaScript usada para criar interfaces de usuário, ou seja, o que as pessoas veem e interagem em um site ou aplicativo. Ele é fácil de adicionar a projetos já existentes e é muito utilizado para construir sites que funcionam como uma única página (SPAs)."
```